include/GafferScene/InteractiveRender.h

```diff
@@ -120,7 +122,9 @@ class GAFFERSCENE_API InteractiveRender : public Gaffer::ComputeNode
 		void messagesChanged();
 		static void messagesChangedUI();
 
-		void plugDirtied( const Gaffer::Plug *plug );
+		void plugSet( const Gaffer::Plug *plug );
+
+
```

Comment:
Remove excess whitespace.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/commit/a29fa8721bedf84c9282a780367b63e47ca0a2b1